jsonrpc/server.go

```diff
@@ -354,9 +361,9 @@ func (s *Server) handleRequest(ctx context.Context, req *request) (*response, er
 
 	if errAny := tuple[1].Interface(); !isNil(errAny) {
 		res.Error = errAny.(*Error)
+		s.failedRequests.WithLabelValues(req.Method).Inc()
```

Comment:
Hmmm, we should probably include metrics for invalid requests (e.g., that fail deserialization) as well. I think this should be good for now, though.